CLI.md

```diff
@@ -452,6 +452,8 @@ View or update the resource control policy
 * `--write-operation <WRITE_OPERATION>` — Set the price per write operation
 * `--byte-read <BYTE_READ>` — Set the price per byte read
 * `--byte-written <BYTE_WRITTEN>` — Set the price per byte written
+* `--blob-byte-read <BLOB_BYTE_READ>` — Set he price to read a blob, per byte
+* `--blob-byte-published <BLOB_BYTE_PUBLISHED>` — The price to publish a blob, per byte
```

Comment:
Why are these two not the same as `--byte-read` and `--byte-written`?

Reply:
Also, I'm confused - the argument is called `blob byte read` (note absence of _price_ word), the comment says it sets the price (per byte) AND there's another setting called `blob byte read PRICE`.